### PR TITLE
FEAT(graph): get graph data from /graph router.

### PR DESCRIPTION
**Pull request by dozheiny**
get graph data from /graph router with bs4 library
and write models of graph data.
also, add the requirements.txt file for installation without using pip (manual)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+rich~=12.4.4
+requests~=2.28.0
+bonbast~=0.0.5
+setuptools~=60.2.0
+bs4~=0.0.1
```
Comment (SamadiPour) [inline requirements.txt]: We don't need this

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+beautifulsoup4~=4.11.1
```
Comment (SamadiPour) [inline requirements.txt]: We shouldn't add this one here.
Delete requirements and add this to pyproject.toml

File: src/bonbast/data.py [hunk 3]
```diff
@@ -118,3 +118,51 @@ def to_json(self) -> dict:
                 'price': self.price,
             }
         }
+
+
+class Graph:
+    """ Graph model
+    """
+    VALUES = {
```
Comment (SamadiPour) [inline src/bonbast/data.py]: These are the same as currencies. We don't want to have duplicates.
Use what we have in the currencies.

File: src/bonbast/server.py [hunk 4]
```diff
@@ -87,3 +90,61 @@ def get_prices_from_api(token, dont_raise_error=False):
...
+        'cookie': 'cookieconsent_status=true; st_bb=0',
+        'origin': 'https://bonbast.com',
+        'referer': 'https://bonbast.com/',
+        'sec-ch-ua': '".Not/A)Brand";v="99", "Google Chrome";v="103", "Chromium";v="103"',
+        'sec-ch-ua-mobile': '?1',
+        'sec-ch-ua-platform': '"Android"',
+        'sec-fetch-dest': 'empty',
+        'sec-fetch-mode': 'cors',
+        'sec-fetch-site': 'same-origin',
+        'user-agent': 'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) '
```
Comment (SamadiPour) [inline src/bonbast/server.py]: extract `user-agent` value as a parameter. We are using it in 3 places.

File: src/bonbast/server.py [hunk 5]
```diff
@@ -87,3 +90,61 @@ def get_prices_from_api(token, dont_raise_error=False):
         raise SystemExit(err)
 
     return r.json()
+
+
+def get_graph_data(start_date, end_date, currency, dont_raise_error=False):
```
Comment (SamadiPour) [inline src/bonbast/server.py]: Please add parameters and return type

**Review by SamadiPour — CHANGES_REQUESTED** [review]

File: src/bonbast/server.py (hunk 5 above)
Comment (dozheiny) [reply src/bonbast/server.py]: ok, i will do your requests and commit again.

**Review by SamadiPour — APPROVED** [review]

**Pull request merged**